bike-erp/src/test/Inventory/__snapshots__/Inventory.test.tsx.snap

```diff
@@ -13,279 +13,7 @@ exports[`Inventory matches the snapshot 1`] = `
   </div>
```

Comment:
Is this the old snapshot? Will a new one be uploaded?

Reply:
I created a new snapshot to take into account the change